backend/db_admin/simple_viewer.py

```
@@ -0,0 +1,238 @@
+def view_table(table_name):
+    """View table data"""
+    conn = get_db_connection()
+    cursor = conn.cursor()
+    
+    # Get column info
+    cursor.execute(f"PRAGMA table_info({table_name})")
+    columns = [col[1] for col in cursor.fetchall()]
+    
+    # Get data
+    limit = request.args.get('limit', 50)
+    cursor.execute(f"SELECT * FROM {table_name} LIMIT {limit}")
```

Ruff (0.13.1)

Possible SQL injection vector through string-based query construction

(S608)